Storage/test.cc

```diff
@@ -1613,6 +1613,15 @@ TEST(TransactionalStorage, ReplicationViaHTTP) {
   const auto master_storage_file_remover = current::FileSystem::ScopedRmFile(master_storage_file_name);
   Storage master_storage(master_storage_file_name);
   master_storage.ExposeRawLogViaHTTP(FLAGS_transactional_storage_test_port, "/raw_log");
+  const std::string base_url = Printf("http://localhost:%d/raw_log", FLAGS_transactional_storage_test_port);
+
+  // Check that the Storage is available via HTTP.
+  {
+    const auto result = HTTP(GET(base_url + "?schema=json"));
+    EXPECT_EQ(200, static_cast<int>(result.code));
+    const auto result2 = HTTP(GET(base_url + "/schema.json"));
+    EXPECT_EQ(200, static_cast<int>(result2.code));
```

Comment:
:+1: could you also add an EXPECT_EQ that the `.body` responses of two GET requests match?